css/hoot-style.css

```diff
@@ -122,6 +122,16 @@ path.shadow.tag-hoot.area:not(.hover).activeReviewFeature2:not(.selected) {
   stroke-width:2;
 }
 
+g.point.edited .shadow {
+    stroke: #FFDF00 !important;
```

Comment:
This may be lazy development on my part.  There is another css class 'tag-hoot-_map-id_' that is used to set the color on features to reflect the dataset color (source1, source2, or merged) and it is taking precedence over the 'edited' style.